x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/datafeed_modal.tsx

```diff
@@ -219,26 +234,28 @@ export const DatafeedModal: FC<DatafeedModalProps> = ({ jobId, end, onClose }) =
             <EuiFlexGroup direction="column">
               <EuiFlexItem grow={false}>
                 <EuiFlexGroup justifyContent="spaceBetween">
-                  <EuiFlexItem grow={false}>
```

Comment:
With this component removed, there is a lot of unused space at the bottom of the modal. Could the chart height be increased to maximise the space e.g. from `300` to about `380`?

Reply:
Updated in 9dbeded8112f701d102898ecf028914bf244dbe2